src/Logging/JUnit/TestSuite.php

```diff
@@ -66,6 +66,15 @@ class TestSuite
      */
     public $cases = [];
 
+    /**
+     * @param string $name
+     * @param int $tests
+     * @param int $assertions
+     * @param int $failures
+     * @param int $skipped
+     * @param float $time
+     * @param ?string $file
```

Comment:
I assume this means `@param string|null $file`.
Is there any spec for this? At least in the [phpdoc doc](https://phpdoc.org/docs/latest/guides/types.html) I can't find anything about it